File: test/aidbox_sdk/snapshots/dotnet/hl7-fhir-r4-core/List.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class List : DomainResource, IResource
{
    public string? Date { get; set; }
    public Base.ResourceReference? Encounter { get; set; }
    public Base.CodeableConcept? OrderedBy { get; set; }
    public required string Mode { get; set; }
    public Base.ResourceReference? Source { get; set; }
    public string? Title { get; set; }
    public Base.Annotation[]? Note { get; set; }
    public Base.CodeableConcept? EmptyReason { get; set; }
    public required string Status { get; set; }
    public Base.CodeableConcept? Code { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public List_Entry[]? Entry { get; set; }
    public Base.ResourceReference? Subject { get; set; }

    public class List_Entry : BackboneElement
    {
        public string? Date { get; set; }
        public Base.CodeableConcept? Flag { get; set; }
        public required Base.ResourceReference Item { get; set; }
        public bool? Deleted { get; set; }
    }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/List.py
from pydantic import *
from typing import Optional, List
from ..base import *

class List_Entry(BackboneElement):
    date: Optional[str] = None
    flag: Optional[CodeableConcept] = None
    item: Reference
    deleted: Optional[bool] = None

class List(DomainResource):
    date: Optional[str] = None
    encounter: Optional[Reference] = None
    ordered_by: Optional[CodeableConcept] = None
    mode: str
    source: Optional[Reference] = None
    title: Optional[str] = None
    note: Optional[List[Annotation]] = None
    empty_reason: Optional[CodeableConcept] = None
    status: str
    code: Optional[CodeableConcept] = None
    identifier: Optional[List[Identifier]] = None
    entry: Optional[List[List_Entry]] = None
    subject: Optional[Reference] = None